app/assets/images/payment_icons/bread.svg

```diff
@@ -0,0 +1 @@
4-.903-.172-.223-.418-.39-.742-.5zm-2.71-2.094h1.288c.418 0 .719.063.903.195.183.13.277.328.277.602 0 .293-.102.508-.3.64-.2.133-.49.2-.872.2h-1.297zm2.241 4.06c-.199.151-.543.225-1.03.225h-1.212v-1.734h1.23c.446 0 .774.066.993.203.215.133.324.348.324.648 0 .29-.101.508-.305.657zm3.879-3.529c-.27.2-.453.504-.55.914l-.149-1.168h-.918v4.977h1.117v-2.3c0-.388.04-.696.117-.923a.806.806 0 01.43-.492c.102-.05.23-.09.38-.117a3.29 3.29 0 01.519-.04h.496v-1.15h-.34c-.465 0-.836.101-1.102.3zm6.207.395a2.04 2.04 0 00-.761-.563c-.309-.14-.676-.207-1.102-.207-.52 0-.969.106-1.348.32a2.198 2.198 0 00-.878.9c-.106.194-.184.41-.235.636-.05.23-.078.476-.078.742v.121c0 .766.207 1.38.625 1.832.414.453 1.04.68 1.879.68.652 0 1.18-.145 1.582-.426.398-.28.672-.676.816-1.18h-1.218a1.03 1.03 0 01-.41.496c-.184.114-.426.172-.727.172-.41 0-.738-.11-.985-.328-.242-.219-.39-.53-.437-.937h3.848v-.528a3.22 3.22 0 00-.137-.926 2.362 2.362 0 00-.434-.804zm-3.265 1.367c.062-.39.215-.695.457-.914.242-.215.558-.324.953-.324.398 0 .703.113.914.332.21.222.336.527.375.906zm8.218-1.922a2.17 2.17 0 00-.562-.164 4.542 4.542 0 00-.727-.05c-.71 0-1.27.171-1.671.515-.407.34-.63.797-.668 1.363h1.168c.02-.32.125-.562.32-.738.2-.172.5-.258.91-.258.387 0 .66.06.816.18a.59.59 0 01.23.496.43.43 0 01-.288.426c-.192.086-.512.156-.957.215-.313.046-.57.09-.77.128-.203.04-.375.086-.52.141-.07.02-.14.05-.214.086a1.99 1.99 0 00-.211.121 1.349 1.349 0 00-.477.496 1.435 1.435 0 00-.175.723c0 .406.14.73.414.972.273.243.68.364 1.215.364.515 0 .94-.11 1.277-.32a2.11 2.11 0 00.762-.805l.12.996h.919v-3.363c0-.356-.075-.665-.215-.926-.14-.262-.371-.46-.696-.598zm-.171 2.633c0 .238-.067.473-.196.707a1.571 1.571 0 01-1.41.8c-.258 0-.453-.054-.582-.155-.129-.106-.195-.262-.195-.47a.605.605 0 01.293-.52 1.51 1.51 0 01.402-.175c.156-.043.383-.09.684-.136a4.3 4.3 0 00.59-.13c.171-.05.308-.116.414-.19zm5.714-4.418v2.547a1.647 1.647 0 00-.628-.715c-.274-.168-.614-.254-1.024-.254-.422 0-.797.094-1.117.285a1.884 1.884 0 00-.754.84 2.45 2.45 0 00-.21.625 4.09 4.09 0 00-.067.75v.125c0 .836.191 1.473.578 1.91.386.442.898.66 1.535.66.445 0 .809-.105 1.094-.32a1.85 1.85 0 00.629-.824l.12 1.04h.946V8.305zm.028 4.191c0 .493-.125.883-.367 1.172-.246.285-.579.43-1 .43a1.19 1.19 0 01-.93-.402c-.242-.266-.363-.668-.363-1.2v-.093c0-.496.117-.883.347-1.16.23-.278.559-.418.989-.418.41 0 .73.144.968.433.239.29.356.672.356 1.152zm0 0" fill="#0f2233"/><path d="M4.953 6.888c-1.343 0-2.305.402-2.305.81 0 .16.106.241.282.288.379.101 3.175.843 3.363.902.223.066 1.344.36 1.344 1.492v-.004S2.219 8.908 1.84 8.826c-.266-.06-.465-.008-.602.128C.906 9.286.5 10.814.5 11.9c0 2.894 1.77 4.988 4.453 4.988 1.344 0 2.305-.402 2.305-.809 0-.16-.106-.242-.281-.289-.38-.1-3.176-.85-3.36-.902-.187-.047-1.347-.36-1.347-1.496l5.796 1.56c.266.057.465.006.602-.13.332-.332.738-1.86.738-2.946 0-2.894-1.77-4.988-4.453-4.988zm-1.336 5.21c-.176-.046-1.347-.366-1.347-1.5l4.027 1.083c.137.035 1.34.371 1.34 1.496 0 0-3.848-1.035-4.02-1.078zm0 0" fill="#5156ea"/></svg>
```

Comment:
We are missing the borders: [video demonstration](https://screenshot.click/07-49-62cn8-ryfqj.mp4)

Please follow the [contributions](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md) guideline

You can follow this [example icon](https://github.com/activemerchant/payment_icons/blob/master/app/assets/images/payment_icons/visa.svg) as a reference.